services/application.js

```diff
@@ -197,29 +210,21 @@ class Application {
 
         const baseConditions = {...statusCondition, ...programNameCondition, ...studyNameCondition, ...submitterNameCondition};
         return (() => {
-            switch (userRole) {
-                case ROLES.ADMIN:
-                case ROLES.FEDERAL_LEAD:
-                case ROLES.DATA_COMMONS_PERSONNEL:
+            switch (true) {
```

Comment:
Using switch on a true value is weird, please just use if-else to test this.

Reply:
Changed both data submission/submission request.